pkg/api/types.go

```diff
@@ -673,6 +673,9 @@ type GlusterfsVolumeSource struct {
 	// Required: EndpointsName is the endpoint name that details Glusterfs topology
 	EndpointsName string `json:"endpoints"`
 
+	// Optional: Servers is the gluster server list as a string seperated by a comma.
+	Servers string `json:"servers,omitempty"`
```

Comment:
@erictune Thanks for the mention ! In most of the cases, its just 'one' server like NFS . If at all, there are more than one, there is a tendency to mention it in comma separate list. Let me check with other folks what they think.  I wll act accordingly.